src/components/header/_macro.spec.js

```diff
@@ -828,155 +643,143 @@ describe('macro: header', () => {
+          ariaLimitedResults: 'Type more characters to improve your search',
+          minChars: 3,
+          language: 'en-gb',
+          ariaMinChars: 'Enter 3 or more characters for suggestions.',
+          ariaNResults: 'There are {n} suggestions available.',
+          ariaOneResult: 'There is one suggestion available.',
+          ariaResultsLabel: 'Country suggestions',
+          ariaYouHaveSelected: 'You have selected',
+          containerClasses: 'ons-autosuggest--header',
+          id: 'ons-site-search',
+          input: {
+            accessiblePlaceholder: true,
+            autocomplete: 'off',
+            classes: 'ons-input-search ons-input-search--icon',
+            label: {
+              classes: 'ons-u-pl-m ons-label--white',
+              id: 'ons-site-search-label',
+              text: 'label',
+            },
+          },
+          instructions: 'Use up and down keys to navigate.',
+          moreResults: 'Continue entering to improve suggestions',
+          noResults: 'No suggestions found.',
+          resultsTitle: 'Suggestions',
+          typeMore: 'Continue entering to get suggestions',
+        });
+      });
+    });
+    describe('WHEN: language is set', () => {
+      const mockData = [
+        { en: 'England' },
+        { en: 'Wales' },
+        { en: 'Scotland' },
+        { en: 'United States of America' },
+        { en: 'United States Virgin Islands' },
+        { en: 'Åland Islands' },
+      ];
+      const $ = cheerio.load(
         renderComponent('header', {
           ...EXAMPLE_HEADER_NAVIGATION_WITH_SITESEARCHAUTOSUGGEST,
           autosuggestData: mockData,
         }),
       );
+      test('THEN: rendered search component accesses set language', () => {
+        $('.ons-js-autosuggest-input').val('Eng');
+        const autosuggestElement = $('.ons-js-autosuggest').attr('data-lang');
+        expect(autosuggestElement).toBe('en-gb');
+      });
     });
+    describe('WHEN: user enters text', () => {
+      let $; // Initialize a Cheerio instance
 
-    it('does not show suggestions when input length < minimum characters', () => {
-      $('.ons-js-autosuggest-input').val('En');
-
-      setTimeout(() => {
-        const suggestionCount = $('.ons-autosuggest__option').length;
-        expect(suggestionCount).toBe(0);
-      }, 20);
-    });
-
-    it('shows suggestions when input length >= minimum characters', () => {
-      $('.ons-js-autosuggest-input').val('Eng');
-
-      setTimeout(() => {
-        const suggestionCount = $('.ons-autosuggest__option').length;
-        expect(suggestionCount).toBe(1);
-      }, 20);
-    });
-
-    it('gets the language if set', () => {
-      $('.ons-js-autosuggest-input').val('Eng');
+      const mockData = [
+        { en: 'England' },
+        { en: 'Wales' },
+        { en: 'Scotland' },
+        { en: 'United States of America' },
+        { en: 'United States Virgin Islands' },
+        { en: 'Åland Islands' },
+      ];
```

Comment:
Could set this a level up and then we wouldn't need to set it twice

Reply:
done!